cli/create.go

```diff
@@ -44,6 +44,7 @@ func (cc *CreateCommand) addFlags() {
 	flagSet.BoolVarP(&cc.tty, "tty", "t", false, "Allocate a tty device")
 	flagSet.StringSliceVarP(&cc.volume, "volume", "v", nil, "Bind mount volumes to container")
 	flagSet.StringVar(&cc.runtime, "runtime", "", "Specify oci runtime")
+	flagSet.StringSliceVarP(&cc.labels, "--label", "l", nil, "Set label for a container")
```

Comment:
s/label/labels

Reply:
the usage is "pouch create -l a=b -l 1=2 xxxxx", so I think we should use "label", not "labels".

Reply:
Different aspects. I think both are reasonable. 🦁